subscriptions.yaml

```diff
@@ -73,7 +73,7 @@ Enterprise Administration:
   - Ubuntu Pro
   - Credentials and certs
 
-Cloud and Infrastructure:
+Cloud & Infrastructure:
```

Comment:
The [copy guidelines](https://discourse.canonical.com/t/copy-reviews/129) state we should use `and` instead of & unless its part of a brand name.

Reply:
This change was based on a request from one of the stakeholders to align how the heading looked. Which should we follow in this instance?